### PR TITLE
merge list group and describe group to get meta group

### PR DESCRIPTION
**Pull request by lianxmfor**
This PR combines list group and describe group to get meta group.

related to #651

### CODE REVIEW

File: oomcli/cmd/get_meta_group.go [hunk 1]
```diff
@@ -100,14 +110,14 @@ func printGroupsInASCIITable(groups types.GroupList) error {
 }
 
 func groupHeader() []string {
-	return []string{"GroupName", "GroupID", "EntityName", "Description", "OnlineRevisionID", "CreateTime", "ModifyTime"}
+	return []string{"ID", "NAME", "ENTITY", "DESCRIPTION", "ONLINE-REVISION-ID", "CREATE-TIME", "MODIFYTIME"}
```
Comment (jinghancc) [inline oomcli/cmd/get_meta_group.go:113]: MODIFYTIME -> MODIFY-TIME

**Review by jinghancc — APPROVED** [review]
LGTM

File: oomcli/test/test_update_group.sh [hunk 2]
```diff
@@ -9,11 +9,9 @@ import_sample > /dev/null
...
-Entity:           device
-Description:      new description
-OnlineRevisionID: <NULL>
+NAME,ENTITY,DESCRIPTION
+phone,device,new description
 '
-actual=$(oomcli describe group phone)
-ignore() { grep -Ev '^(CreateTime|ModifyTime)' <<<"$1"; }
+ignore() { cut -d ',' -f 2-4 <<<"$1"; }
+actual=$(oomcli get meta group phone -o csv)
```
Comment (wfxr) [inline oomcli/test/test_update_group.sh:16]: Suggest list complete output in `expected` and do the same filter logic on `expected` and `actual` so we can known what the output looks like and what was filtered actually.

File: oomcli/cmd/get_meta_group.go (hunk 1 above)
Comment (lianxmfor) [reply oomcli/cmd/get_meta_group.go]: good catch! done.

File: oomcli/test/test_update_group.sh (hunk 2 above)
Comment (lianxmfor) [reply oomcli/test/test_update_group.sh:16]: sounds good. updated.

File: oomcli/test/test_apply_group.sh [hunk 3]
```diff
@@ -30,13 +30,13 @@ EOF
 oomcli apply -f "$TMPFILE"
 
 group_expected='
-GroupName,GroupID,EntityName,Description
-device,1,user,description
-account,2,user,description
+ID,NAME,ENTITY,DESCRIPTION,NONLINE-REVISION-ID,CREATE-TIME,MODIFY-TIME
```
Comment (wfxr) [inline oomcli/test/test_apply_group.sh:33]: `NONLINE-REVISION-ID` => `ONLINE-REVISION-ID`

File: oomcli/test/test_apply_group.sh [hunk 4]
```diff
@@ -67,12 +67,12 @@ EOF
 oomcli apply -f "$TMPFILE"
 
 group_expected='
-GroupName,GroupID,EntityName,Description
-device,1,user,description
+ID,NAME,ENTITY,DESCRIPTION,NONLINE-REVISION-ID,CREATE-TIME,MODIFY-TIME
```
Comment (wfxr) [inline oomcli/test/test_apply_group.sh:70]: `NONLINE-REVISION-ID` => `ONLINE-REVISION-ID`

File: oomcli/test/test_update_group.sh [hunk 5]
```diff
@@ -9,11 +9,9 @@ import_sample > /dev/null
 case='oomcli update group works'
 oomcli update group phone --description "new description"
 expected='
-Name:             phone
-Entity:           device
-Description:      new description
-OnlineRevisionID: <NULL>
+ID,NAME,ENTITY,DESCRIPTION,NONLINE-REVISION-ID,CREATE-TIME,MODIFY-TIME
```
Comment (wfxr) [inline oomcli/test/test_update_group.sh:12]: `NONLINE-REVISION-ID` => `ONLINE-REVISION-ID`

File: oomcli/test/test_apply_group.sh (hunk 4 above)
Comment (lianxmfor) [reply oomcli/test/test_apply_group.sh]: fixed

File: oomcli/test/test_update_group.sh (hunk 5 above)
Comment (lianxmfor) [reply oomcli/test/test_update_group.sh]: fixed

File: oomcli/test/test_apply_group.sh (hunk 3 above)
Comment (lianxmfor) [reply oomcli/test/test_apply_group.sh]: fixed

**Review by wfxr — APPROVED** [review]
LGTM

**Pull request merged**